### PR TITLE
Cleanup of f-strings and imports

### PR DESCRIPTION
**Pull request by marksmayo**
### Before submitting a *pull request* make sure you have:
- [X ] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [X ] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [X ] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check all of the following options that apply:
- [X ] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [X ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [X ] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

</details>

### CODE REVIEW

**Comment by pukkandan** [comment]
1. https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes
2. Style-only changes to extractors are not welcome as it only makes it harder for me to merge with upstream
	* common, generic, youtube as well as any file not in ytdl are allowed to be modified
4. https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#quotes
5. Many of the fstrings are worse than their original. These should be reverted.
	* What formatting tool did you use? [My last style refactoring](86e5f3ed2e6e71eb81ea4c9e26288f16119ffd0c) was with `pyupgrade` which is quite conservative, but even then, I had to cherrypick specific fstring changes
	* Basically, don't make fstring with inner quotes or `{}` unless you are willing to manually evaluate the merit of each
6. Imports should not be split with `\`

**Comment by pzhlkj6612** [comment]
We now already have the "UP031" and "UP032" rules for checking formatted strings since #10149:

https://github.com/yt-dlp/yt-dlp/blob/41be32e78c3845000dbac188ffb90ea3ea7c4dfa/pyproject.toml#L234

- [printf-string-formatting (UP031) - Ruff](https://docs.astral.sh/ruff/rules/printf-string-formatting/)
- [f-string (UP032) - Ruff](https://docs.astral.sh/ruff/rules/f-string/)

Test:

```Python
a = 'b %s' % 42        # UP031
a = 'b {}'.format(42)  # UP032
a = f'b {42}'          # fine

```

<br>

Regarding imports, we now also have the "I001" rule:

https://github.com/yt-dlp/yt-dlp/blob/41be32e78c3845000dbac188ffb90ea3ea7c4dfa/pyproject.toml#L230

- [unsorted-imports (I001) - Ruff](https://docs.astral.sh/ruff/rules/unsorted-imports/)

.
